app/scheme.go

```diff
@@ -171,7 +173,8 @@ func (a *App) SchemesIterator(batchSize int) func() []*model.Scheme {
 	offset := 0
 	return func() []*model.Scheme {
 		var result store.StoreResult
```

Comment:
This line can be removed, and you can do the assignment in the line below with `:=`